spec/models/event_spec.rb

```diff
@@ -16,9 +16,15 @@
   describe 'attribute validations' do
     it 'does not validate when starts_at is not defined' do
       event = Event.new(starts_at: nil)
-      expect(event).to_not be_valid
+      expect(event).to be_invalid
       expect(event.errors.messages.keys).to include :starts_at
     end
+
+    it 'does not validate when location_id is not defined' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.